docker/beta/rc.local

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# AcousticBrainz rc.local
+# Enables a specific runit service depending on what the
+# CONTAINER_NAME variable is set to
+
+
+if [ "${CONTAINER_NAME}" = "acousticbrainz-web-beta" ]
```

Comment:
Imho this env variable should be named `CONTAINER_ROLE`, container names are set externally, and can change, this is why we register "services" in prod.
But, since those roles are exclusive, the correct approach would be to have separate containers (and images).

Reply:
I'm happy to call this CONTAINER_ROLE, but CONTAINER_NAME is what's currently used in AB and other python-brainzes (e.g. https://github.com/metabrainz/listenbrainz-server/blob/6cd87707144cac9bafa2529f9c93e3c3f48ff5cc/docker/prod/uwsgi/uwsgi.service). What other changes are needed (in deploy scripts?) to ensure that a new environment variable gets set?

What do you mean about separating containers and images? We already run each of these services as a separate container (web, hl extractor, dataset evaluator, cron)

Reply:
I've changed this to CONTAINER_ROLE and opened a sister PR in docker-server-configs (https://github.com/metabrainz/docker-server-configs/pull/78)